File: src/easy/ReverseString.swift
func reverseString(_ s: inout [Character]) {
  var left = 0
  var right = s.endIndex - 1
  var char: Character
  while left < right {
    char = s[left]
    s[left] = s[right]
    s[right] = char
    left += 1
    right -= 1
  }
}

File: lib/easy/reverse_string.dart
void reverseString(List<String> s) {
  String char = "";
  for (int left = 0, right = s.length - 1; left < right; left++, right--) {
    char = s[left];
    s[left] = s[right];
    s[right] = char;
  }
}
